pkg/util/kubernetes/hostinfo/tags.go

```diff
@@ -37,6 +30,28 @@ func GetTags() ([]string, error) {
 	return extractTags(nodeLabels, labelsToTags), nil
 }
 
+func getDefaultLabelsToTags() map[string]string {
+	return map[string]string{
+		"kubernetes.io/role": "kube_node_role",
+	}
+}
+
+func getLabelsToTags() map[string]string {
+	labelsToTags := getDefaultLabelsToTags()
+	for k, v := range config.Datadog.GetStringMapString("kubernetes_node_labels_as_tags") {
+		labelsToTags[k] = v
+	}
+	if len(labelsToTags) == 0 {
+		return nil
+	}
+	// viper lower-cases map keys from yaml, but not from envvars
+	for label, value := range labelsToTags {
+		delete(labelsToTags, label)
+		labelsToTags[strings.ToLower(label)] = value
+	}
```

Comment:
can we do this in the first loop ? line https://github.com/DataDog/datadog-agent/pull/4594/files#diff-a6d3ead700aba07975ffa3e831b332dcR41